src/main/webapp/manage-availability.html

```diff
@@ -0,0 +1,189 @@
+    limitations under the License.
+-->
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Your Availability</title>
+        <link rel="stylesheet" type="text/css" href="general.css">
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    </head>
+    <body onload="getAvailabilityManage(), displayLoginLogoutLink()">
+        <a href="./homepage.html"><p>Homepage</p></a>
+
+        <div class="top-right">
+            <a id="login-url"><p id="login" style="display:none">Login with Google</p></a>
+            <a id="logout-url"><p id="logout" style="display:none">Logout</p></a>
+        </div>
+
+        <div class="container-fluid">
+            <h1>Manage Your Availability</h1>
+            <h2>Delete available timeslots or add new ones.</h2>
+        </div>
+
+        <div class="container-fluid">
+            <ul class="list-group" id="timeslots"></ul>
+        </div>
+
+        <div class="container-fluid">
+            <h2>Add a new available timeslot.</h2>
+
+            <form class="form-inline" action="/addAvailability" method="POST">
+                <textarea class="form-control" name="tutorEmail" id="tutorEmail" style="display: none;"></textarea>
+
+                <label class="my-1 mr-2" for="startHour startMinute">From:</label>
+                <select class="custom-select form-control my-1 mr-sm-2" name="startHour" id="startHour" required>
+                    <option value="">select</option>
+                    <option value="0">00</option>
+                    <option value="1">01</option>
+                    <option value="2">02</option>
+                    <option value="3">03</option>
+                    <option value="4">04</option>
+                    <option value="5">05</option>
+                    <option value="6">06</option>
+                    <option value="7">07</option>
+                    <option value="8">08</option>
+                    <option value="9">09</option>
+                    <option value="10">10</option>
+                    <option value="11">11</option>
+                    <option value="12">12</option>
+                    <option value="13">13</option>
+                    <option value="14">14</option>
+                    <option value="15">15</option>
+                    <option value="16">16</option>
+                    <option value="17">17</option>
+                    <option value="18">18</option>
+                    <option value="19">19</option>
+                    <option value="20">20</option>
+                    <option value="21">21</option>
+                    <option value="22">22</option>
+                    <option value="23">23</option>
```

Comment:
We can definitely worry about this later but it's something to think about: Should we have hours in AM/PM? Or maybe we could do some kind of location based thing so it displays 24 hour time if you're accessing the webapp from a non-American country.

Reply:
Yeah that would be better. For this first version I think it would be okay though. But I can definitely do this next week,